cmake/FindFreetype.cmake

```diff
@@ -78,21 +77,13 @@ find_path(Freetype_OLD_INCLUDE_DIR
     PATH_SUFFIXES freetype2
     NO_DEFAULT_PATH
 )
-
-# get version from freetype.h
-find_file(Freetype_HEADER
-    NAMES freetype.h
-    PATH_SUFFIXES freetype
-    PATHS ${Freetype_OLD_INCLUDE_DIR}
+libfind_version_n_header(Freetype
```

Comment:
Rewrite `find_file` and `get_version_from_n_defines` with `libfind_version_n_header`